ensembldl.sh

```diff
@@ -29,7 +29,7 @@ USAGE
 
 DESCRIPTION
 
-  download ENSEMBL data set
+  download ENSEMBL prefix dataset
```

Comment:
This is the description of the tool, not the command line usage. The usage is directly above and contains the prefix. Please remove this change.

Reply:
done